applications/CompressiblePotentialFlowApplication/custom_processes/compute_nodal_value_process.h

```diff
@@ -0,0 +1,178 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marc Nunez, Inigo Lopez based on R.Rossi and V.Mataix work
+//
+//
+
+#if !defined(KRATOS_COMPUTE_NODAL_VALUE_PROCESS_INCLUDED)
+#define  KRATOS_COMPUTE_NODAL_VALUE_PROCESS_INCLUDED
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+
+class KRATOS_API(CompressiblePotentialFlowApplication) ComputeNodalValueProcess
```

Comment:
```suggestion
class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) ComputeNodalValueProcess
```
This was wrongly introduced by me, windows wont compile otherwise